src/Form/Form.js

```diff
@@ -38,7 +38,6 @@ class Form {
 
         this.touched = new Set()
         this.listeners = new Set()
-        this.tooltip = null
```

Comment:
agreed this is much cleaner, tooltip manipulation should remain within the device